UI/window-dock-youtube-app.cpp

```diff
@@ -0,0 +1,392 @@
+#include <QUuid>
+
+#include "window-basic-main.hpp"
+#include "youtube-api-wrappers.hpp"
+#include "window-dock-youtube-app.hpp"
+
+#include "ui-config.h"
+
+#ifdef YOUTUBE_WEBAPP_PLACEHOLDER
+constexpr const char *YOUTUBE_WEBAPP_PLACEHOLDER_URL =
+	YOUTUBE_WEBAPP_PLACEHOLDER;
+#else
+constexpr const char *YOUTUBE_WEBAPP_PLACEHOLDER_URL =
+	"https://studio.youtube.com/live/channel/UC/console?kc=OBS";
+#endif
+
+#ifdef YOUTUBE_WEBAPP_ADDRESS
+constexpr const char *YOUTUBE_WEBAPP_ADDRESS_URL = YOUTUBE_WEBAPP_ADDRESS;
+#else
+constexpr const char *YOUTUBE_WEBAPP_ADDRESS_URL =
+	"https://studio.youtube.com/live/channel/%1/console?kc=OBS";
+#endif
+
+constexpr const char *BROADCAST_CREATED = "BROADCAST_CREATED";
```

Comment:
Use `static constexpr` to ensure the variable is only initialised once and also only available in this compilation unit (it will probably moved into the `TEXT` area of the binary).

Indeed there are only a few situations where a `constexpr` should not be `static`.

Reply:
Done.